index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <link rel="stylesheet" href='style.css' type="text/css">
+  <title>Tic Tac Toe Project</title>
+</head>
+<body onload="">
+	<div class="grid">
+		<div class="row">
+			<div id="box-0" onclick="addMarker(this)" class="boxes"></div>
```

Comment:
Nice use of "this" keyword. I would like you to talk about "this" with the class. It can be confusing but very useful.